content/en/blog/_posts/2021-06-21-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: 'Updates to Ingress Nginx with network v1'
+date: 2021-06-21
+slug: update-with-ingress-nginx
+---
+
+**Authors:** James Strong, Ricardo Katz
+
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The networking.k8s.io API group is no
+different. The upcoming Kubernetes 1.22 release will remove several deprecated APIs
+that are relevant to networking:
+
+- the `networking.k8s.io/v1beta1` API version of [IngressClass](/docs/concepts/services-networking/ingress/#ingress-class)
+- all beta versions of [Ingress](/docs/concepts/services-networking/ingress/): `extensions/v1beta1` and `networking.k8s.io/v1beta1`
+
+On a v1.22 Kubernetes cluster, you'll be able to access Ingress and IngressClass
+objects through the stable (v1) APIs, but access via their beta APIs won't be possible.
+This change has been in
+in discussion since
+[2017](https://github.com/kubernetes/kubernetes/issues/43214),
+[2019](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) with 
+1.16 Kubernetes API deprecations,and most recently in 
+KEP-1453: 
+[Graduate Ingress API to GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api#122).
+
+During community meetings, the networking Special Interest Group has decided to continue 
+supporting Kubernetes versions older than 1.22 with Ingress-NGINX version 0.47.0. 
+Support for Ingress-NGINX will continue for six months after Kubernetes 1.22 
+is released. Any additional bug fixes and CVEs for Ingress-NGINX will be 
+addressed on a need-by-need basis.
+
+Ingress-NGINX will have separate branches and releases of Ingress-NGINX to 
+support this model, mirroring the Kubernetes project process. Future 
+releases of the Ingress-NGINX project will track and support the latest 
+versions of Kubernetes.
+
+{{< table caption="Ingress NGINX supported version with Kubernetes Versions" >}}
+Kubernetes Version  | Ingress-NGINX version | Support Notes
+:-------------------|:----------------------|:------------
+`1.22`              | TBD                   | On Going
+`1.21`              | `v0.47.0`             | On Going but only CVE and crashes
+`1.20`              | `v0.47.0`             | On Going but only CVE and crashes
+`1.19`              | `v0.47.0`             | Support will drop 6 months after 1.22 release
```

Comment:
Is this accurate?
```suggestion
v1.22              | ≥ v.0.47.0     | New features, plus bug fixes.
v1.21              | v0.47.x        | Bugfixes only, and just for security issues or crashes. No end-of-support date announced.
v1.20              | v0.47.x        | Bugfixes only, and just  for security issues or crashes. No end-of-support date announced.
v1.19              | v0.47.x        | Bugfixes only, and just  for security issues or crashes. Fixes only provided until 6 months after Kubernetes v1.22.0 is released.
```

I changed `v0.47.0` to `v0.47.x` because I assume there would be a new patch release, rather than yanking `v0.47.0` and replacing it with different code.